recipes/onetbb/all/conanfile.py

```diff
@@ -52,7 +53,9 @@ def _tbbbind_hwloc_version(self):
 
     @property
     def _tbbbind_supported(self):
-        return self.settings.os != "Macos" or Version(self.version) >= "2021.11.0"
+        if not is_apple_os(self):
+            return True
+        return self.settings.os == "Macos" and Version(self.version) >= "2021.11.0"
```

Comment:
```suggestion
        if is_apple_os(self):
            return return self.settings.os == "Macos" and Version(self.version) >= "2021.11.0"
        return True
```
Seems easier to read to me. Just a suggestion.